src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,147 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    static final int DEFAULT_INITIAL_LENGTH = 1 << 4;
+    static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private int actualLoadFactorNumber;
```

Comment:
Class variable are hard to maintain and keep safe. Better to avoid them them if possible
This one can be calculated from other ones
```suggestion
```

Reply:
Not fixed

Reply:
?